docs/index.markdown

```diff
@@ -0,0 +1,160 @@
+---
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: default
+exclude: true
+---
+
+# Introduction
+
+The Blossom extension is a "Single Sign-On" solution for Web3 applications. It provides security and helps dApps to avoid implementing account management. By managing dApp permissions, users have complete control of the dApps they are using.
+
+# Prerequisites
+
+In order to function properly, the Blossom extension requires the Swarm extension to be installed and configured.
+
+Open the [Google Web Store page](https://chrome.google.com/webstore/detail/ethereum-swarm-extension/afpgelfcknfbbfnipnomfdbbnbbemnia){:target="\_blank"} and install the Swarm extension first.
+
+The Swarm extension provides security context for dApps that are loaded from bzz links.
+
+## Swarm Extension Configuration
+
+Configure your Swarm extension, by setting the correct URL of your Bee node. Later the Blossom will use this URL as storage.
+
+If you are running a local Bee node on the default port, then no changes are needed. Otherwise update it to correct URL.
+
+![Swarm Extension URL config](/assets/img/swarm-extension-bee-url.png){:width="70%"}
+
+If you don't have a running Bee node, you can use the FDS gateway:
+
+```
+https://bee-1.fairdatasociety.org
+```
+
```

Comment:
If you use a public gateway, you do not have control over data persistence and lose some privacy, as the gateway sees the traffic.

Reply:
I will add this as a note.